src/components/Icons/User.js

```diff
@@ -25,14 +27,12 @@ UserIcon.displayName = "UserIcon";
 
 UserIcon.propTypes = {
   children: PropTypes.node,
-  color: PropTypes.string,
-  size: PropTypes.number
+  color: PropTypes.string
 };
 
 UserIcon.defaultProps = {
   children: null,
-  color: "currentColor",
-  size: 24
```

Comment:
I fear the hard-coding of width/height for UserIcon could be a potential breaking change, given how sensitive the icon is for UX. How do you feel about leaving the `size` prop, but converting it to a type of string with a default value of "24"?

Reply:
Okay, I'll leave the type as number, because this might bring warning in code that is already using the user icon. According to design team, this icon should have only one size - 24, so let's say that size is deprecated and will plan removing it in the future.

Reply:
Alright